src/index.js

```diff
@@ -116,6 +116,12 @@ Dropzone.propTypes = {
    * Maximum file size (in bytes)
    */
   maxSize: PropTypes.number,
+  /**
+   * Maximum accepted number of files
+   * The default value is 0 which means there is no limitation in the accepted number of files
+   * The minimum accepted value is 2
```

Comment:
Why would 2 be min? I'd say it can be anything. If it's 0 or less, then this prop is disabled, if it's 1 or more then it's enabled.

Reply:
it doesn't make sense because it will be enabled when  multiple is enabled so 0 or 1 is not correct as an accepted value

Reply:
Why does it have to be tied to `multiple`? Even if `multiple` is false, the user could drag several files, one at a time, up to the pre-defined maximum value. Allowing a maximum value of 1 seems logical. 

Also, thank you for taking the time to implement this feature! I see it has been requested for a long time and I too would like it for a feature I'm currently building!

Reply:
@Morteza-Jenabzadeh `multiple` does not enforce how many files you need/can to drop. You can drop 1, 2, etc. So I see using `maxFiles` as a way to limit how many files a user can drop, be it 1, 2 or more.

Therefore, I think allowing >= 1 makes sense.